src/components/AvatarGroup/AvatarGroupCounterTooltipContainer.jsx

```diff
@@ -42,24 +38,29 @@ const AvatarGroupCounterTooltipContainer = ({
     tooltipContentContainerRef,
     focusPrevPlaceholderRef,
     focusNextPlaceholderRef,
-    setShouldUpdate,
-    setIsTooltipVisible
+    setIsKeyboardTooltipVisible,
+    isKeyboardTooltipVisible
   });
 
-  if (!avatars?.length && !counterTooltipCustomProps?.content) {
-    return children;
-  }
+  // Tooltip props
+  const onHide = useCallback(() => {
+    setIsKeyboardTooltipVisible(false);
+  }, []);
+  const showTrigger = useMemo(() => [Dialog.hideShowTriggers.MOUSE_ENTER], []);
+  const hideTrigger = useMemo(() => [Dialog.hideShowTriggers.MOUSE_LEAVE], []);
```

Comment:
Why we can't just define these consts outside of component declaration without using memo?

Reply:
Or request here a memo... I thought that in this case there is not matter for giving to dialog component a new instance each render. wdyt @orrgottlieb ?